src/wallet/gtest/test_wallet.cpp

```diff
@@ -137,6 +137,7 @@ TEST(WalletTests, SetupDatadirLocationRunAsFirstTest) {
     mapArgs["-datadir"] = pathTemp.string();
 }
 
+/*
 TEST(WalletTests, WalletNetworkSerialization) {
```

Comment:
Consider this instead:
```rust
    GTEST_SKIP() << "Skipping WalletNetworkSerialization because it cannot be run in parallel.";
```
This ensures that the test is still compiled, so is less likely to bitrot in ways that make it more difficult to fix when we get around to that. `GTEST_SKIP` was added in https://github.com/google/googletest/pull/1544 ; if we don't have it then we can [prepend `DISABLED_` to the test name](https://stackoverflow.com/a/7208119/393146) instead.